tests/fixtures/test_brief.py

```diff
@@ -560,3 +561,208 @@ def test_404_is_returned_when_deleting_a_missing_brief(client, overview_briefs):
+
+
+def test_rfx_brief_create_success(client, overview_users, rfx_lot):
+    res = client.post('/2/login', data=json.dumps({
+        'emailAddress': 'me@digital.gov.au', 'password': 'test'
+    }), content_type='application/json')
+    assert res.status_code == 200
+
+    res = client.post('/2/brief/rfx', content_type='application/json')
+    assert res.status_code == 200
+
+    response = json.loads(res.data)
+    assert response['id'] == 1
+
+
+def test_rfx_publish_success_2_days_correct_dates(client, overview_users, rfx_lot, rfx_brief):
+    res = client.post('/2/login', data=json.dumps({
+        'emailAddress': 'me@digital.gov.au', 'password': 'test'
+    }), content_type='application/json')
+    assert res.status_code == 200
+
+    res = client.patch('/2/brief/1', content_type='application/json', data=json.dumps({
+        'publish': True,
+        'closedAt': pendulum.today(tz='Australia/Sydney').add(days=2).format('%Y-%m-%d')
+    }))
+    assert res.status_code == 200
+    response = json.loads(res.data)
+    assert response['closedAt'] == pendulum.today().add(days=2).format('%Y-%m-%d')
+    question_closing_date = pendulum.instance(workday(pendulum.today(), 1)).format('%Y-%m-%d')
+    if question_closing_date > response['closedAt']:
+        question_closing_date = pendulum.today().format('%Y-%m-%d')
+    assert response['dates']['questions_closing_date'] == question_closing_date
+
+
+def test_rfx_publish_failure_next_day(client, overview_users, rfx_lot, rfx_brief):
+    res = client.post('/2/login', data=json.dumps({
+        'emailAddress': 'me@digital.gov.au', 'password': 'test'
+    }), content_type='application/json')
+    assert res.status_code == 200
+
+    res = client.patch('/2/brief/1', content_type='application/json', data=json.dumps({
+        'publish': True,
+        'closedAt': pendulum.today().add(days=1).format('%Y-%m-%d')
+    }))
+    assert res.status_code == 400
+
+
+def test_rfx_publish_success_three_days_correct_dates(client, overview_users, rfx_lot, rfx_brief):
+    res = client.post('/2/login', data=json.dumps({
+        'emailAddress': 'me@digital.gov.au', 'password': 'test'
+    }), content_type='application/json')
+    assert res.status_code == 200
+
+    res = client.patch('/2/brief/1', content_type='application/json', data=json.dumps({
+        'publish': True,
+        'closedAt': pendulum.today().add(days=3).format('%Y-%m-%d')
+    }))
+    assert res.status_code == 200
+    response = json.loads(res.data)
+    assert response['closedAt'] == pendulum.today().add(days=3).format('%Y-%m-%d')
+    question_closing_date = pendulum.instance(workday(pendulum.today(), 2)).format('%Y-%m-%d')
+    if question_closing_date > response['closedAt']:
+        question_closing_date = response['closedAt']
+    assert response['dates']['questions_closing_date'] == question_closing_date
+
+
+def test_rfx_publish_success_under_one_week_correct_dates(client, overview_users, rfx_lot, rfx_brief):
```

Comment:
Does it make sense to reduce repetition here by merging this test with the one above with `parametrize`?

Reply:
I think the `questions_closing_date` asserting is different enough between each publish success test that it may not reduce much repetition or, if it does, may introduce more complexity to the tests.